### ISSUE TITLE
Unify `Text` and `TextLink` accessibility

### ISSUE THREAD

**Issue by PavelHolec** [kiwicom/orbit-swiftui#451]
Text and `Text` containing `TextLink`s should have the same accessibility properties the same way as the native versions.
